### PR TITLE
fix: Change size of front page image to fit lower screen resolutions

### PR DESCRIPTION
**Pull request by marcusstigelid**
- Adds new files site-header.html and index.html, which are duplicated from the theme in order to change some CSS classes
- Updates the image file with a different size and crop
- Moves and adds upon the "Welcome" text on the front page to above the title to make the posts more visible

### CODE REVIEW

File: layouts/partials/site-header.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{{ $featured_image := partial "func/GetFeaturedImage.html" . }}
+{{ if $featured_image }}
+  {{/* Trimming the slash and adding absURL make sure the image works no matter where our site lives */}}
+  <header class="cover bg-top" style="background-image: url('{{ $featured_image }}');">
+    <div class="{{ .Site.Params.cover_dimming_class | default "bg-black-60" }}">
+      {{ partial "site-navigation.html" .}}
+      <div class="tc-l pv4 pv5-l ph3 ph4-ns">
```
Comment (marcusstigelid) [inline layouts/partials/site-header.html:7]: Note: This is the line that has been changed from the default theme, specifically the "pv5-l" class which by default is "pv6-l"

**Comment by marcusstigelid** [comment]
This is the effect of the changes as seen on my Macbook:

Current design:
blog_current


New design
blog_new

File: content/_index.md [hunk 2]
```diff
@@ -5,5 +5,3 @@ cascade:
   featured_image: '/sparvagn.jpg'
   
 ---
-
-Welcome
```
Comment (gadapunkfun) [inline content/_index.md:9]: Why was this removed?

Comment (marcusstigelid) [reply content/_index.md:9]: I felt it was a bit unnecessary and removing it makes the posts more visible (as seen in the screenshots). But if we want to keep the text (and maybe expand it to be more than just "Welcome") I can add it again.

Comment (gadapunkfun) [reply content/_index.md:9]: Suggestion, move the welcome to the main title and add,
<p>Welcome to our communal</p>
<h1>Tech Culture Business Blog</h1>
?

Comment (marcusstigelid) [reply content/_index.md:9]: By main title do you mean the text that is in the center of the image? If so that could work, I am just trying to figure out how I can insert a newline, adding the standard <br /> tag does not work it seems.

**Comment by marcusstigelid** [comment]
This is how it looks now after the latest change:
Screenshot 2023-06-13 at 13 00 35

File: content/_index.md (hunk 2 above)
Comment (marcusstigelid) [reply content/_index.md:9]: This has now been added to site-header.html

File: layouts/index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{{ define "main" }}
+ <article class="cf ph3 ph5-l pv3 pv2-l f4 tc-l center measure-wide lh-copy {{ $.Param "text_color" | default "mid-gray" }}">
```
Comment (marcusstigelid) [inline layouts/index.html:2]: This file has been copied from the theme and the only thing that has been changed is the "pv2-l" class which previously was "pv4-l"

**Review by gadapunkfun — APPROVED** [review]
Looks good! 👍🏼
